AudioSystem/AudioSystem.cs

```diff
@@ -77,7 +77,7 @@ public bool Update(float dt)
 
 				timer += dt;
 				channel.volume = Mathf.Lerp(from, to, timer / duration);
-				if (timer >= duration)
```

Comment:
I'm guessing this change should not have gone in this PR

Reply:
Weird that it's shown in here when this was merged in first:
https://github.com/dubit/duck/pull/69

Reply:
__**FancyGitHubFeature™**__